erpnext/selling/doctype/quotation/quotation.py

```diff
@@ -148,6 +148,10 @@ def set_missing_values(source, target):
 	def update_item(obj, target, source_parent):
 		target.stock_qty = flt(obj.qty) * flt(obj.conversion_factor)
 
+	def update_payment_schedule(obj, target, source_parent):
+		if target.due_date < getdate(nowdate()):
```

Comment:
```suggestion
		if getdate(target.due_date) < getdate(nowdate()):
```

Its always a good practise to wrap a date object in `get_date`.